docs/guides/building/windows.md

```diff
@@ -24,40 +24,104 @@ tauri build --target i686-pc-windows-msvc
 By default Rust only installs toolchains for your machine's target, so you need to install the 32-bit Windows toolchain first: `rustup target add i686-pc-windows-msvc`. You can get a full list of Rust targets by running `rustup target list`.
 :::
 
-## Using a Fixed Version of the Webview2 Runtime
+## Supporting Windows 7
 
-By default, the Tauri installer downloads and installs the Webview2 Runtime if it is not already installed (on Windows 11, WebView2 is preinstalled).
+By default the Microsoft Installer does not work on Windows 7 because it needs to download the Webview2 bootstrapper if the runtime is not installed, which might fail if TLS 1.2 is not enabled in the operating system. Tauri includes an option to embed the Webview2 bootstrapper, see the [section below](#embedding-the-webview2-bootstrapper).
```

Comment:
```suggestion
By default the Microsoft Installer does not work on Windows 7 because it needs to download the Webview2 bootstrapper if not installed (which might fail if TLS 1.2 is not enabled in the operating system). Tauri includes an option to embed the Webview2 bootstrapper (see the [Embedding the Webview2 Bootstrapper](#embedding-the-webview2-bootstrapper) section below).
```